### ISSUE TITLE
iframe needs `allow-popups allow-popups-to-escape-sandbox`

### ISSUE THREAD

**Issue by walid0925** [GibbsConsulting/django-plotly-dash#411]
seeing an error of the variety
```
Blocked opening '<my_url>' in a new window because the request was made in a
sandboxed frame whose 'allow-popups' permission is not set.
```
when trying to open a page in a new tab via a button inside the app

it seems as though these sandbox permissions are needed [here](https://github.com/GibbsConsulting/django-plotly-dash/blob/4ac19feda1084a16ace0196b4f7379bfeee264fc/django_plotly_dash/templates/django_plotly_dash/plotly_app.html#L2)

**Comment by GibbsConsulting** [comment]
This is needed in both templates. Created #412 to address sandbox properties as this keeps cropping up.

@walid0925 what version of `django-plotly-dash` were you using, and what browser(s) are stumbling over this particular permission?

**Comment by walid0925** [comment]
> This is needed in both templates. Created #412 to address sandbox properties as this keeps cropping up.
> 
> @walid0925 what version of `django-plotly-dash` were you using, and what browser(s) are stumbling over this particular permission?

thank you! v1.7.1 and noticed navigation issues in both Chrome and Safari. only Chrome showed the error in console

**Comment by GibbsConsulting** [comment]
Addressed in v2.1.0
